Magic/src/main/java/com/elmakers/mine/bukkit/action/BaseShopAction.java

```diff
@@ -532,6 +544,7 @@ public void prepare(CastContext context, ConfigurationSection parameters) {
         costScale = parameters.getDouble("scale", 1);
         filterBound = parameters.getBoolean("filter_bound", false);
         putInHand = parameters.getBoolean("put_in_hand", true);
+        //requiredSkillapiClass = parameters.getString("skillapi_class", null);
```

Comment:
Commented out?
`#getString` returns null when nothing is available, so you can drop the `, null`
https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/configuration/ConfigurationSection.html#getString-java.lang.String-